crates/stages/src/stages/execution.rs

```diff
@@ -86,19 +87,24 @@ impl<EF: ExecutorFactory> ExecutionStage<EF> {
     // TODO: This should be in the block provider trait once we consolidate
     // SharedDatabase/Transaction
     fn read_block_with_senders<DB: Database>(
```

Comment:
Reminder on this TODO, easy block provider function

Reply:
follow-up PR